bytes.go

```diff
@@ -26,8 +30,8 @@ func (b *bytesFrom) Dist(to []byte) int {
 				min = m[y+1]
 			}
 		}
-		if min >= b.maxCost {
-			return b.maxCost
+		if min > b.maxCost {
```

Comment:
In other words: to match how `levenshtein_less_equal` works in postgresql and because we suspect that otherwise, each client of this package will need to increment `maxDistance` before the call (`FromBytes(word, maxDistance + 1)`), just as we do.